maxim/logger/portkey/portkey.py

```
@@ -0,0 +1,184 @@
+from typing import Optional, Union
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)